app/views/admin/project_steps/_project_step.html.slim

```diff
@@ -9,6 +9,15 @@ div.panel.step style="border-color: #{step.border_color}" data-step-id="#{step.i
 
   = render "admin/project_steps/duplicate_step_modal", step: step if policy(step).duplicate?
 
+  - days_shifted ||= 0
+  - if days_shifted > 0
+    = render "admin/project_steps/shift_subsequent_confirm_modal", step: step,
+      days_shifted: days_shifted, subsequent_count: subsequent_count
+    javascript:
+      $(function () {
+        $("#step-shift-subsequent-confirm-modal-#{step.id}").modal("show");
+      });
```

Comment:
Sure.  Just to confirm, you're suggesting to move not literally just the JS block but the whole 'days shifted' block, lines 12-19?